### PR TITLE
Allow to delete my evals

### PR DESCRIPTION
**Pull request by reinier-millo**
- [x] Should be merged after #27

### CODE REVIEW

File: client/app/components/resultItem.tsx [hunk 1]
```diff
@@ -59,37 +89,113 @@ export default function ResultItem({
...
-    router.push('/');
+    router.push("/");
+  };
+
+  const deleteEvalOnConfirm = async () => {
+    onClose();
+    console.log("deleting eval", id);
+    deleteEval(id, accessToken)
+      .then(() => {
+        toast({
```
Comment (belindamo) [inline client/app/components/resultItem.tsx:100]: description "Eval deleted!"
duration 8000

File: client/app/components/resultItem.tsx [hunk 2]
```diff
@@ -59,37 +89,113 @@ export default function ResultItem({
...
-        textAlign='start'
-        _hover={{ backgroundColor: 'gray.100', cursor: 'pointer' }}
+        textAlign="start"
+        _hover={{ backgroundColor: "gray.100", cursor: "pointer" }}
       >
         <HStack>
-          <Avatar size='xs' name={mainAuthor?.username ?? 'Unknown'} src={mainAuthor?.avatar ?? 'https://www.svgrepo.com/show/448095/person-circle.svg'} />
+          <Avatar
+            size="xs"
+            name={mainAuthor?.username ?? "Unknown"}
```
Comment (belindamo) [inline client/app/components/resultItem.tsx:127]: If mainAuthor is not there, then I think this whole Avatar and name shouldn't show up

**Review by belindamo — COMMENTED** [review]
only reviewed the last commit, the review for the other commits is at https://github.com/openevals/evals/pull/27

File: client/app/components/resultItem.tsx (hunk 2 above)
Comment (reinier-millo) [reply client/app/components/resultItem.tsx:127]: I added a validation to show only if main author is valid. In the practice main author should be always defined.

**Review by justinlinw — APPROVED** [review]
Lgtm

**Pull request merged**